ansible_rulebook/schema/ruleset_schema.json

```diff
@@ -51,6 +51,28 @@
                 ]
             }
         },
+        "throttle": {
+            "type": "object",
+	    "anyOf": [
```

Comment:
Correct me if I am wrong, but I don't think there are any cases where you would use `once_within` and `once_after` in a single throttle configuration? If this is true then I think the constraint here should be `oneOf`.

Reply:
Also it's bad indented. But I see multiple places where we have a bad indentation making it more difficult to read. We can fix everything in a separated PR

Reply:
@ttuffin Fixed